config/stub_templates/ruby/write.rb.jinja

```diff
@@ -1,6 +1,14 @@
+{# 
+    NOTE: CG prints the comments to the right if they are only one line long, 
+    and above if multiline. We only do above, regardless of size, for the moment. 
```

Comment:
Probably for the best, there are also situations where CG does not print comments at all iirc